streams/src/main/java/org/apache/kafka/streams/kstream/TimeWindowedKStream.java

```diff
@@ -217,23 +226,29 @@ KTable<Windowed<K>, Long> count(final Named named,
                                     final Materialized<K, Long, WindowStore<Bytes, byte[]>> materialized);
 
     /**
-     * Aggregate the values of records in this stream by the grouped key and defined windows.
```

Comment:
The diff starting here is just "bad" and can be "ignored". I just re-ordered method, to align method order across different classes. There is no actual change below.